go.mod

```
@@ -20,62 +20,47 @@ require (
 	google.golang.org/protobuf v1.30.0
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.26.1
-	k8s.io/apimachinery v0.26.1
-	k8s.io/client-go v0.26.1
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
-	sigs.k8s.io/controller-runtime v0.13.1
-	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230326190753-3b0376c24e4f
-	sigs.k8s.io/controller-tools v0.11.3
-)
-
-replace (
-	k8s.io/api => k8s.io/api v0.24.12
-	k8s.io/apimachinery => k8s.io/apimachinery v0.24.12
-	k8s.io/client-go => k8s.io/client-go v0.24.12
+	k8s.io/api v0.27.3
+	k8s.io/apimachinery v0.27.3
+	k8s.io/client-go v0.27.3
+	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	sigs.k8s.io/controller-runtime v0.15.0
+	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230702162849-a23f3603e1dc
```

Optionally, pin this to the same version as controller-runtime.
```suggestion
sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230523032821-116a1b831fff // v0.15.0
```